.circleci/deploy.sh

```diff
@@ -44,35 +45,62 @@ export CLOUDSDK_CORE_DISABLE_PROMPTS=1
 docker login -u _json_key -p "$(cat ${HOME}/gcloud-service-key.json)" https://gcr.io
 
-echo "Installing Azure CLI"
-apt update
-apt install -y apt-transport-https
-echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ stretch main" | tee /etc/apt/sources.list.d/azure-cli.list
-curl -L https://packages.microsoft.com/keys/microsoft.asc | apt-key add
-apt update
-apt install -y azure-cli
+echo "Configuring Docker Hub configuration"
+echo ${DOCKER_PASS} | docker login -u ${DOCKER_USER} --password-stdin
+
+echo "Configuring Quay configuration"
+echo ${QUAY_PASS} | docker login ghcr.io -u ${QUAY_USER} --password-stdin
 
 echo "Building the tiller image"
 make docker-build VERSION="${VERSION}"
 
+# Image is pushed to GitHub container repository (ghcr.io),
+# GCR, Docker Hub, and Quay.
+echo "Pushing image to ghcr.io"
+docker push "ghcr.io/helm/tiller:${VERSION}"
+
 echo "Pushing image to gcr.io"
+docker tag "ghcr.io/helm/tiller:${VERSION}" "gcr.io/kubernetes-helm/tiller:${VERSION}"
 docker push "gcr.io/kubernetes-helm/tiller:${VERSION}"
 
-echo "Building helm binaries"
-make build-cross
-make dist checksum VERSION="${VERSION}"
+echo "Pushing image to Docker Hub"
+docker tag "ghcr.io/helm/tiller:${VERSION}" "helmpack/tiller:${VERSION}"
+docker push "helmpack/tiller:${VERSION}"
+
+echo "Pushing image to Quay"
+docker tag "ghcr.io/helm/tiller:${VERSION}" "quay.io/helmpack/tiller:${VERSION}"
+docker push "quay.io/helmpack/tiller:${VERSION}"
+
+# Canary version is used with helm init --canary-image flag.
+# Does not push canary binary which is Helm v3.
+ifneq ($(VERSION),"canary")
+  echo "Installing Azure CLI"
+  apt update
+  apt install -y apt-transport-https
+  echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ stretch main" | tee /etc/apt/sources.list.d/azure-cli.list
+  curl -L https://packages.microsoft.com/keys/microsoft.asc | apt-key add
+  apt update
+  apt install -y azure-cli
+
+  echo "Building helm binaries"
+  make build-cross
+  make dist checksum VERSION="${VERSION}"
 
-echo "Pushing binaries to gs bucket"
-${HOME}/google-cloud-sdk/bin/gsutil cp ./_dist/* "gs://${PROJECT_NAME}"
+  echo "Pushing binaries to gs bucket"
+  ${HOME}/google-cloud-sdk/bin/gsutil cp ./_dist/* "gs://${PROJECT_NAME}"
 
-echo "Pushing binaries to Azure"
-az storage blob upload-batch -s _dist/ -d "$AZURE_STORAGE_CONTAINER_NAME" --pattern 'helm-*' --connection-string "$AZURE_STORAGE_CONNECTION_STRING"
+  echo "Pushing binaries to Azure"
+  az storage blob upload-batch -s _dist/ -d "$AZURE_STORAGE_CONTAINER_NAME" --pattern 'helm-*' --connection-string "$AZURE_STORAGE_CONNECTION_STRING"
 
-echo "Pushing KEYS file to Azure"
-az storage blob upload -f "KEYS" -n "KEYS" -c "$AZURE_STORAGE_CONTAINER_NAME" --connection-string "$AZURE_STORAGE_CONNECTION_STRING"
+  echo "Pushing KEYS file to Azure"
+  az storage blob upload -f "KEYS" -n "KEYS" -c "$AZURE_STORAGE_CONTAINER_NAME" --connection-string "$AZURE_STORAGE_CONNECTION_STRING"
+endif
```

Comment:
shellify. also adding newline at EOF

```suggestion
fi

```

Reply:
Fixed. thanks